samcli/cli/cli_config_file.py

```diff
@@ -245,13 +245,9 @@ def decorator_customize_config_file(f):
     """
     config_file_attrs = {}
     config_file_param_decls = ("--config-file",)
-    config_file_attrs["help"] = (
-        "The path and file name of the configuration file containing default parameter values to use. "
-        "Its default value is 'samconfig.toml' in project directory. For more information about configuration files, "
-        "see: "
-        "https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-config.html."
-    )
+    config_file_attrs["help"] = "Configuration file containing default parameter values."
```

Comment:
Should we still keep link to documentation about configuration?

Reply:
We still do I'm moving it to a separate option heading as its cluttered otherwise.

Reply:
and `sam config docs` is going to address this.